assets/js/modules/analytics/components/settings/SettingsView.js

```diff
@@ -63,6 +64,10 @@ export default function SettingsView() {
 		path: `/a${ accountID }w${ internalWebPropertyID }p${ profileID }/admin/view/settings`,
 	} ) );
 
+	const editViewSettingsGA4URL = useSelect( ( select ) => select( STORE_NAME ).getServiceURL( {
```

Comment:
GA4 doesn't have Views (Profiles) so we should update this to be more specific to a data stream. Data streams are also GA4-specific so we also don't need to include GA4 in the name to disambiguate either 🙂 
 ```suggestion
	const editDataStreamSettingsURL = useSelect( ( select ) => select( STORE_NAME ).getServiceURL( {
```

Reply:
Done :+1: